routers/repo/issue.go

```diff
@@ -1074,8 +1076,38 @@ func UpdateIssueContent(ctx *context.Context) {
 		return
 	}
 
+	files := ctx.QueryStrings("files[]")
+	for i := 0; i < len(issue.Attachments); i++ {
+		if !util.IsStringInSlice(issue.Attachments[i].UUID, files) {
```

Comment:
Use without ! and `continue` and move out if below

Reply:
Update, @lafriks thank you for the review! 😄